book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
-This notation is equivalent to a multi-state representation of the recurrent events process, where each event number represents a different state.
-In @fig-recurrent-events, right panel, this is refered to as "Clock forward" approach.
+For illustration, consider again the `prothr` data set (@tbl-ms-data-prothr), with possible transitions summarized in @fig-multi-state-prothr-states.
+In contrast to the illness-death model in @fig-multi-state, back transitions are possible and some subjects already start in the "abnormal" state at the beginning of the study.
 
-| ID| $k$ | $t_{start}$| $t_{stop}$|$\delta$|$gap$|
-|--:|---:|---:|------:|--:|-----:|
-|  1|1|  0 | 219|   1| 219| 
-|  1|2|219 | 373|   1| 154| 
-|  1|3|373 | 414|   0|  41| 
-|  7|1|  0 | 292|   1| 292| 
-|  7|2|292 | 364|   0|  72| 
-: Subset of the `cgd`  [@pkgsurvival] time-to-event dataset.
-Rows are individual observation units ($ID$), $k$ is the $k$th event occurrence, $t_{start}$ and $t_{stop}$ mark the beginning and end of the "at risk time" for the $k$th event occurence, $\delta$ is the event indicator and $gap = t_{stop} - t_{start}$. {#tbl-cgd-data}
+![Transition graph for the liver cirrhosis patients.](Figures/survival/multi-state-prothr-states.png){#fig-multi-state-prothr-states fig-alt="Transition probabilities for the prothrombin data example."}
 
-For recurrent events data without competing (terminal) events, the "Clock reset" approach (@fig-recurrent-events, left panel) can be used.
-Instead of representing the data in the start-stop notation, the gap time is calculated instead (see fourth column in @tbl-cgd-data).
-In this representation, we only consider how long the subject was at risk for the $k$th event, not at which time (on the calendar time scale).
-It has the advantage that standard methods for analysis of single-event data can be used for modeling.
-However, the resulting dependency between multiple data points from one subject (at risk for multiple events at the same time) needs to be taken into account in the modeling step.
 
-![Illustration of transitions in the recurrent events setting. Left: "Clock reset" representation of recurrent events. Each time an event occurs, the clock is reset to 0 and we once again wait for occurence of an event (of the same type); Right: "Clock forward" representation. A subject transitions from 1st, to 2nd, etc. to $K$th event occurence](Figures/survival/recurrent-events.png){#fig-recurrent-events fig-alt="Schematic illustration of recurrent events transitions."}
+Figure @fig-multi-state-prothr shows the transition probabilities for the four possible transitions over time for subjects who received treatment and placebo, respectively.
+In this example back-transitions are possible, therefore, in contrast to the cumulative incidence functions in the competing risks setting, transition probabilities (to transient states) are not monotonously increasing over time.
```

Comment:
Hm this feels slightly off to me because CIF is a CDF type function that is monotonously increasing by definition whereas transition probabilities is a vector of probabilities over time so there's no intuitive reason why they'd ever be monotonically increasing/decreasing? LMK if i misunderstood

Reply:
I think the other thing, similar to my comment above, is if we should also plot H or S? I think the concern could be that we've essentially introduced a very fundamentally different paradigm so even if it's usually modelled with P, if you've shown equation H_NA,le then surely it follows you can get S_NA,le -- would plotting that be interesting? Personally I'd be interested in knowing the probability of surviving one transition to the next, but if that's abnormal then ignore me!